tests/kernel/mem_heap/shared_multi_heap/boards/qemu_cortex_a53.overlay

```diff
@@ -5,30 +5,32 @@
  */
 
 / {
-	reserved-memory {
-		compatible = "reserved-memory";
-		#address-cells = <1>;
```

Comment:
Just a question, is there a mechanism to make sure the default value of `#address-cells` and `#size-cells` are 2 in aarch64 and correspondingly 1 in aarch32

Reply:
not that I'm aware of. Each overlay file should specify that accordingly.

Reply:
OK, as long as it works :)